packages/core/index.test-d.ts

```diff
@@ -4,9 +4,18 @@ import {
   APIGatewayProxyEvent,
   APIGatewayProxyResult,
   Context,
-  Handler as LambdaHandler
+  Handler as AWSLambdaHandler
 } from 'aws-lambda'
 
+type ModifyReturnType<T, NewReturn> = T extends (
```

Comment:
I think I landed on an implementation. This gives the ability to still depend on `AWSLambdaHandler` from aws and add an additional return type